jsamchineme/producthunt-backend
# Create new product

### Description
This change allows us to create a new product. It features an image upload

### How to test
- pull down the branch
- run the following commands
   - `npm run heroku-postbuild`
   - `npm run start`
- using postman, make a POST request to the endpoint "http://localhost:3000/products", set the request body type to `form-data` and supply values for the following fields
  - productName: "sample product Name" (text)
  - description: "sample description" (text)
  - image: File (select a file)

- check the response returned as matching this sample
```
{
    "data": {
        "id": "2f2dc11c-1ca1-461c-9b40-5ffcdbb88658",
        "productName": "dummy name",
        "description": "dummy description",
        "bannerImage": "http://localhost:3000/uploads/image-1599753461681.png"
    }
}
